src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -222,12 +245,17 @@
                                 } else {
                                     FileUtils.copyURLToFile(new URL(index, line + ".hpi"), new File(plugins, line + ".jpi"));
                                 }
-                                // TODO add method to disable a plugin (e.g. to test optional dependencies)
                             }
                         }
                     }
+                    for (String p : disabledPlugins) {
```

Comment:
I am assuming that we just do not skip copying the plugin as we re-use the plugin directory for all the test that may use a `RealJenkinsRule`?  If not why not just skip them rather than disabling them (and change disable plugins to `withoutPlugins` as that is a better use case at least for operations center things?)

Reply:
> why not just skip [copying] them rather than disabling them

Good point, that would probably work. Will try it…